config/locales/en/admin.yml

```diff
@@ -215,7 +216,7 @@ en:
         title: "Edit phase"
         description_help_text: This text will appear in the header when the phase is active
         duration: "Phase's duration"
-        duration_description: "The period of time this phase will be active."
+        duration_description: "The period of time this phase will be active. Remember that this field is for information only and does not trigger an automatic update of the active phase."
```

Comment:
I was using the form and I read "Phase's duration, blah, blah, blah, start date, end date" :thinking:.

So maybe we could add hints in three places: the one we're using now, under the "start date" label, and under the "end date" label.

Since the first sentence isn't really true and is also a bit redundant because administrators will already expect that when they see the date fields, maybe we could focus on the second one:

```suggestion
        duration_description: "These fields are used for information purposes only and do not trigger an automatic update of the active phase. In order to update it, edit the budget and select the active phase."
```

And then, in both "start date" and "end date", we could add a hint saying "For information purposes only".

We could also add a `<strong>` tag for the "do not trigger an automatic update of the active phase"  text :thinking:. I'm wondering whether we should then modify the styles slightly:

```diff
diff --git a/app/assets/stylesheets/layout.scss b/app/assets/stylesheets/layout.scss
index b4eecfd5f..46036acb7 100644
--- a/app/assets/stylesheets/layout.scss
+++ b/app/assets/stylesheets/layout.scss
@@ -450,6 +450,10 @@ button,
 .help-text {
   line-height: rem-calc(20);
   margin-top: 0;
+
+  strong {
+    font-style: normal;
+  }
 }
 
 .menu-and-content {
```

If you think this is overkill, say so! :wink: